unity/EditorPlugin/Notifications.cs

```diff
@@ -0,0 +1,29 @@
+using System;
+using JetBrains.Rider.Unity.Editor.Utils;
+using JetBrains.Util;
+using UnityEngine;
+
+namespace JetBrains.Rider.Unity.Editor
+{
+  public static class Notifications
+  {
+    public static void ShowAutoSaveNotificationIfAllowed()
+    {
+      if (RiderScriptableSingleton.Instance.AutoSaveWarningShownOnce || PluginSettings.SelectedLoggingLevel < LoggingLevel.WARN) return;
+      
+      var notification =
+       @"AutoSaving files in Rider may trigger recompilation when switching back to Unity Editor, which may wipe live data if the Unity player is running.
+       Consider changing <b>Script Changes While Playing</b> setting in Unity-Preferences-";
```

Comment:
How about:

> Auto save is enabled in Rider. This can cause unwanted recompilation and the loss of current state during play mode.
> Consider changing the *Script Changes While Playing* setting in the {0} preferences page.

With either "General" or "Rider"?